gallery/src/components/NxCombobox/NxComboboxDisabledExample.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useCallback, useRef, useState } from 'react';
+import { filter, map, prepend, range } from 'ramda';
+import { useDebounceCallback } from '@react-hook/debounce';
+import { NxCombobox, DataItem, NX_SEARCH_DROPDOWN_DEBOUNCE_TIME, NxFormGroup }
+  from '@sonatype/react-shared-components';
+
+const items = prepend(
+    { id: 0, displayName: 'Loooooooooooooooooooooooooong Name' },
+    map(i => ({ id: i, displayName: `Item ${i}` }), range(1, 101))
+);
+
+// This function simulates a backend query that takes 3 seconds to return results. In a real implementation
+// this would typically use window.fetch, axios, or a similar REST library rather than querying in-memory data,
+// and typically this would be in another file outside of the react component
+function search(query: string): Promise<DataItem<number>[]> {
```

Comment:
Unlike `NxSearchDropdown`, where every realistic example needed to simulate a backend call, `NxCombobox` has uses in synchronous cases as well (as you've demonstrated in some of the other examples).  That opens the door for us to simplify this disabled example - it doesn't need to pretend to be async.

Reply:
Updated here e5af307e96c19e89ebc39071855bfda658f17b5c.